src/main/java/org/broadinstitute/hellbender/utils/runtime/CapturedStreamOutput.java

```diff
@@ -89,10 +92,11 @@ public boolean isBufferTruncated() {
      *
      * @throws IOException When unable to read or write.
      */
-    public void readAndClose() throws IOException {
+    public boolean read() throws IOException {
```

Comment:
what's the return value mean?

Reply:
Another artifact from a previous implementation. Removed.